doc/releases/release-notes-2.2.rst

```diff
@@ -193,6 +205,7 @@ Boards & SoC Support
    * Google Kukui EC
    * NXP i.MX RT1010 Evaluation Kit
    * Silicon Labs EFM32 Giant Gecko GG11
+   * Silicon Labs EFM32 Jade Gecko
```

Comment:
Sorry, overlooked this line. I would remove this line, since actually no board was added. Only a board already available was modified.

Reply:
This is from efm32pg_stk3402a_jg_defconfig that was added.

```
# SPDX-License-Identifier: Apache-2.0

CONFIG_ARM=y
CONFIG_SOC_SERIES_EFM32JG12B=y
CONFIG_BOARD_EFM32PG_STK3402A_JG=y
CONFIG_CONSOLE=y
CONFIG_UART_CONSOLE=y
CONFIG_SERIAL=y
CONFIG_CORTEX_M_SYSTICK=y
CONFIG_GPIO=y
CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=40000000
CONFIG_CMU_HFCLK_HFXO=y
```

It's a new board target - i think it is fine to list it in the boards, no?

Reply:
We did add a new `-DBOARD=`-able target.

But, does that constitute a new board? I think that is a rather philosophical question ...

Reply:
It makes sense to add this in the release notes since that's something the Zephyr users will actually see immediately, i.e. a new compilation option. 

Also: since we added the corresponding SoC support in the release notes, it is fine, i think, to specify, here, the Board that _enables_ that SoC to be used

Reply:
Ok, i am fine with this reasoning. I just wanted to make sure that everyone understands that a board that was already present is extended for the new SOC.